lib/document.js

```diff
@@ -4,10 +4,14 @@
  * Module dependencies.
  */
 
+
+const DivergentArrayError = require('./error/divergentArray');
```

Comment:
```suggestion
const DivergentArrayError = require('./error/divergentArray');
```

i dont think 2 empty lines are needed